lib/app.js

```diff
@@ -26,6 +27,32 @@ const app = net.createServer(socket => {
       fsPromises.readFile('public/index.html')
         .then((content => socket.write(createResponse({ body: content }))));
     }
+    else if (request.path === '/echo' && request.method === 'POST') {
+      socket.write(createResponse({ body: request.body, contentType: 'text/plain', status: '200 OK' })
+      );
+    }
+
+    else if (request.path === '/red' && request.method === 'GET') {
+      socket.write(createResponse({
+        body: '<h1>red<h1>'
+      })
+      );
+    }
+
+    else if (request.path === '/green' && request.method === 'GET') {
+      socket.write(createResponse({ body: '<h1>green<h1>' })
+      );
+    }
+
+    else if (request.path === '/blue' && request.method === 'GET') {
+      socket.write(createResponse({ body: '<h1>blue<h1>' })
+      );
+    }
+
+    else {
+      socket.end(createResponse({ body: 'Not Found', status: '404 Not Found', contentType: 'text/plain' }));
+    }
+
```

Comment:
You could also use some switch cases if you wanted to, if else statements totally work.